src/pages/tutorial/__tests__/__snapshots__/tutorial.test.js.snap

```diff
@@ -104,7 +104,7 @@ exports[`TutorialPage Component should render the TutorialPage component fulfill
         xs={12}
       >
         <h1
-          className="pf-c-title pf-m-2xl"
+          className="pf-c-title pf-m-2xl integr8ly-header-after-diagram"
```

Comment:
As we're pulling in PF4, you can use the margin modifier class instead of the custom one: `pf-u-mt-xl`.

Reply:
How timely... thanks @mindreeper2420. Updated with pf4 style and remove custom.